tests/unit/test_storage_factory.py

```
@@ -325,3 +325,29 @@ async def test_returns_three_tuple(self, tmp_path: Path) -> None:
+
+class TestOutboxFactory:
+    @pytest.mark.asyncio
+    async def test_factory_returns_outbox_writer_when_async_outbox(self, tmp_path: Path) -> None:
+        """async_outbox モード時、Storage に OutboxWriter が注入される."""
+        db_path = str(tmp_path / "test.db")
+        settings = make_settings(
+            storage_backend="sqlite",
+            sqlite_db_path=db_path,
+            graph_enabled=True,
+            graph_sync_mode="async_outbox",
+            neo4j_password="secret",
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "neo4j_password"

(S106)